cmd/soroban-cli/src/commands/config/secret.rs

```diff
@@ -69,21 +76,21 @@ impl Args {
 
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(untagged)]
-pub enum Secret {
+pub enum SignerKind {
```

Comment:
Yeah I was never happy with this name change. It started because I had the ledger as another case. And it didn't make sense. But it does contain the secret key...